edenai_apis/apis/elevenlabs/elevenlabs_api.py

```
@@ -116,3 +117,48 @@ def audio__text_to_speech(
                 audio=audio, voice_type=1, audio_resource_url=resource_url
             ),
         )
+
+    async def audio__atext_to_speech(
```

Pylint (4.0.2)

[refactor] Too many arguments (10/7)

(R0913)